behave/configuration.py

```diff
@@ -28,6 +28,12 @@
 if six.PY2:
     ConfigParser = configparser.SafeConfigParser
 
+try:
+    import toml
+    _TOML_AVAILABLE = True
+    import json
+except ImportError:
+    _TOML_AVAILABLE = False
```

Comment:
`json` is int he standard library I think we can import it before of the `try` 
```suggestion
import json
try:
    import toml
    _TOML_AVAILABLE = True
except ImportError:
    _TOML_AVAILABLE = False
```